system/Config/View.php

```diff
@@ -88,6 +88,17 @@ class View extends BaseConfig
         'siteURL'           => '\CodeIgniter\View\Plugins::siteURL',
     ];
 
+    /**
+     * View Decorators are class methods that will be run in sequence to
+     * have a chance to alter the generated output just prior to caching
+     * the results.
+     *
+     * All classes must implement CodeIgniter\View\ViewDecorator
+     *
+     * @var array
```

Comment:
This should use `class-string` to match the suggestion in **app/Config/View.php**.